src/sentry/integrations/slack/utils/channel.py

```diff
@@ -99,11 +116,145 @@ def validate_channel_id(name: str, integration_id: int | None, input_channel_id:
         )
 
 
+def get_users_sdk(
+    client: SlackSdkClient, integration: Integration | RpcIntegration
+) -> Iterable[Mapping[str, Any]]:
+    payload = {
+        "exclude_archived": False,
+        "exclude_members": True,
+        "types": "public_channel,private_channel",
+    }
+    try:
+        users_list = client.users_list(limit=SLACK_GET_CHANNEL_ID_PAGE_SIZE, kwargs=payload)
```

Comment:
will replace this after cathy's refactor [pr](https://github.com/getsentry/sentry/pull/73085) gets merged